types/web-resource-inliner/index.d.ts

```diff
@@ -0,0 +1,132 @@
+// Type definitions for web-resource-inliner 4.2
+// Project: https://github.com/jrit/web-resource-inliner#readme
+// Definitions by: pinage404 <https://github.com/pinage404>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export namespace WebResourceInliner {
```

Comment:
I don't think you need this namespace, but it's close.
I think all you need is:
```ts
export type Callback = ...
export interface Options = ...

export function html...
export function css...
```
That should fix the build errors.